test/functional/vimscript/screenchar_spec.lua

```diff
@@ -0,0 +1,77 @@
+local helpers = require('test.functional.helpers')(after_each)
+local clear, eq, meths = helpers.clear, helpers.eq, helpers.meths
+local command, funcs = helpers.command, helpers.funcs
+
+local global_helpers = require('test.helpers')
```

Comment:
can delete this; test.helpers is merged-into test.functional.helpers

Reply:
Done.